src/main/java/core/basesyntax/db/StorageImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageImpl implements Storage {
+    private HashMap<String, Integer> fruitMap;
```

Comment:
The field `fruitMap` should be declared as a Map interface instead of a specific implementation to follow the Dependency Inversion principle from SOLID. Consider using `Map<String, Integer>` for the declaration.